### PR TITLE
prompt: log output of browser open command

### PR DESCRIPTION
**Pull request by landism**
### Problem

At the Tilt terminal prompt, if `browser.Open` produces any stdout/stderr, it gets swallowed, which might make errors like #4435 unnecessarily mysterious (at least, all the examples I saw on google of xdg-open exiting with status 3 also had output explaining why).

### Solution

fork github.com/pkg/browser so that we don't have to use its global stderr/stdout vars and can redirect its output appropriately

### Notes

The original code we had to redirect browser.openurl's output put it on logger.DebugLvl. For darwin, `open` doesn't output anything on success, so InfoLvl has no downside. I don't have easy access to a linux box to see if the same is true for xdg-open, but as long as it doesn't have a lot of output, it seems a small price to pay.

### CODE REVIEW

**Comment by landism** [comment]
(also, after looking at the commit that added it, it made sense that we were setting `browser.Stdout` in the k8s client, but it was sure amusing to find it there!)

File: internal/browser/browser_linux.go [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package browser
+
+func openArgv(url string) []string {
+	return []string{"xdg-open", url}
```
Comment (nicks) [inline internal/browser/browser_linux.go:4]: i think the real upstream version is more complex now?
https://github.com/pkg/browser/blob/master/browser_linux.go#L9

File: internal/browser/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This is a fork of https://github.com/pkg/browser that allows changing stdout/stderr non-globally.
```
Comment (nicks) [inline internal/browser/README.md:1]: have you considered having a formal fork live at github.com/tilt-dev/browser? 

i don't think the changes we want here would be unreasonable upstream

**Review by nicks — APPROVED** [review]

File: internal/browser/browser_linux.go (hunk 1 above)
Comment (landism) [reply internal/browser/browser_linux.go:4]: Ah, I was just grabbing it from `vendor`. I'll update.

File: internal/browser/README.md (hunk 2 above)
Comment (landism) [reply internal/browser/README.md:1]: I figured I'd try an upstream PR and then make a formal fork if that goes poorly, but I guess I need a fork from which to make a PR anyway.

**Pull request merged**